devito/parameters.py

```diff
@@ -91,8 +97,23 @@ def initialize(self):
     'DEVITO_DEBUG_COMPILER': 'debug_compiler',
 }
 
+configuration = Parameters("Devito-Configuration")
+"""The Devito configuration parameters."""
+
+configuration.add('travis_test', 0, [0, 1], lambda i: bool(i))
```

Comment:
`travis_test` one is a leftover and can actually be removed.

Reply:
really? I can drop this entire line? Remind me what was it used for? it's been around for ages (and I think I moved it more than once xD)

Reply:
For context, this was introduced with the FWI tutorial to control runtime on travis. It's been removed with the introduction of `nbval` which allows us to simply skip cells.